### PR TITLE
Release style

### PR DESCRIPTION
**Pull request by TMazenge**
Added common styling to the Release History Page.

### CODE REVIEW

File: src/main/webapp/release_history.html [hunk 1]
```diff
@@ -11,57 +11,106 @@
...
+                    <th scope="col">Launch Date</th>
+                    <th scope="col">Branch</th>
+                    <th scope="col">Minor/Major Bump</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                    <th scope="row">19.3.1</th>
+                    <th>June 4 2020</th>
+                    <th><a href="https://github.com/FirebasePrivate/firebase-android-sdk/tree/M72">M72</th>
```
Comment (eriyezab) [inline src/main/webapp/release_history.html]: This is my bad I didn't see this when I committed this part but I forgot to put a closing \</a> tag in all of these.

File: src/main/webapp/release_history.html [hunk 2]
```diff
@@ -11,57 +11,106 @@
...
+    </div>
+
+    <hr>
+
+    <div class="row justify-content-center">
+        <div class="col-md-8">
+            <div id="release-history" class="table-responsive">
+                <table class="table table-bordered justify-content-center" style="border: 1px solid #ddd !important;">
+                <thead>
+                    <tr>
```
Comment (eriyezab) [inline src/main/webapp/release_history.html:72]: all th tags between tr tags should be indented

File: src/main/webapp/release_history.html [hunk 3]
```diff
@@ -11,57 +11,106 @@
...
+           </div>
+        </div>
+    </div>
+
+    <hr>
+
+    <div class="row justify-content-center">
+        <div class="col-md-8">
+            <div id="release-history" class="table-responsive">
+                <table class="table table-bordered justify-content-center" style="border: 1px solid #ddd !important;">
```
Comment (eriyezab) [inline src/main/webapp/release_history.html:70]: ideally styling should strictly be in css file. refer to https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns

**Review by eriyezab — CHANGES_REQUESTED** [review]
nit: indentation should be 2 spaces not 4 https://google.github.io/styleguide/htmlcssguide.html#Indentation

**Review by ankitaj224 — APPROVED** [review]
Mostly LGTM. I am approving it given we will change the dummy data to read from backend.

Thanks

**Review by eriyezab — APPROVED** [review]

**Pull request merged**